tests/pipelines/lint/test_nextflow_config.py

```diff
@@ -69,88 +65,16 @@ def test_nextflow_config_missing_test_profile_failed(self):
         assert len(result["failed"]) > 0
         assert len(result["warned"]) == 0
 
-    def test_default_values_fail(self):
```

Comment:
Instead of removing this test we should change it to use a different variable. `max_multiqc_email_size`, `publish_dir_mode` or `custom_config_version` would be good candidates